assessments/README.md

```diff
@@ -35,24 +34,20 @@ the assessements.
 
 ## Outcome
 
-Each project assessment will:
+Each CNCF project security assessment will:
```

Comment:
not sure "CNCF" adds anything here... maybe for consistency should be simply "security assessment" throughout the document

Reply:
added language to remove unnecessary CNCF project security assessment but still distinguish a external assessment/review from the work performed by the SIG.